qa/common/sequence_util.py

```diff
@@ -986,18 +986,18 @@ def check_setup(self, model_name):
     def check_status(self, model_name, batch_exec, exec_cnt, infer_cnt):
         start_time = time.time()
         # There is a time window between when responses are returned and statistics are updated.
-        # To prevent intermittent test failure during that window, wait up to 10 seconds for the
+        # To prevent intermittent test failure during that window, wait up to 5 seconds for the
```

Comment:
You can keep it as 10 seconds by increasing sleep time. Might as well have more time for it to pass if we're doing the workaround.

Reply:
Sure, updated.